cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/TopPanel/TopPanel.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../../../../styles/variables.scss";
+
+.profile-detail-view {
+  .detail-view-top-panel {
+    padding: 0 50px;
+    height: 50px;
+    background-color: $grey-08;
+    display: flex;
+    align-items: center;
+    font-size: 18px;
+
+    .detail-view-breadcrumb {
+      .icon-chevron-left {
+        font-size: 7px;
```

Comment:
<img width="1440" alt="screen shot 2018-04-25 at 12 48 35 pm" src="https://user-images.githubusercontent.com/1452845/39269190-08e386e2-4887-11e8-9094-e71d2c31bfeb.png">
We have this pattern already. Is there any reason why this is slightly different? 

<img width="1440" alt="screen shot 2018-04-25 at 12 49 19 pm" src="https://user-images.githubusercontent.com/1452845/39269210-1862a68e-4887-11e8-948e-8ef94ea0f064.png">

Reply:
@ajainarayanan I created the detail page design back in 1/2017, I used orange for clickable links. That pattern has since changed (or it never really took hold) and we have been using blue for a while. We do have some inconsistency in the treatment of some the interactive elements, and links are one of them. :-) 
My vote is to change the "details" page link to blue. I would also change the label from "back" to "control center".

Reply:
Filed a JIRA to change the entity detail pages: https://issues.cask.co/browse/CDAP-13369